apps/meteor/client/components/message/content/attachments/file/GenericFileAttachment.tsx

```diff
@@ -55,7 +55,12 @@ const GenericFileAttachment = ({
 					<MessageGenericPreviewContent
 						thumb={<MessageGenericPreviewIcon name='attachment-file' type={format || getFileExtension(title)} />}
 					>
-						<MessageGenericPreviewTitle externalUrl={getExternalUrl()} onClick={handleTitleClick} data-qa-type='attachment-title-link'>
+						<MessageGenericPreviewTitle
+							download={!!openDocumentViewer}
+							externalUrl={getExternalUrl()}
+							onClick={handleTitleClick}
+							data-qa-type='attachment-title-link'
+						>
```

Comment:
I would put this change in another PR, specific to this change. So we can explain it separately, using a different changeset for that, so we can keep track of the change.